src/library/Uno.Toolkit.Material/Styles/Controls/v2/Divider.xaml

```diff
@@ -2,15 +2,29 @@
 					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
 					xmlns:utu="using:Uno.Toolkit.UI">
 
-	<x:Double x:Key="MaterialDividerHeight">1</x:Double>
 
-	<Style x:Key="MaterialDividerStyle"
-		   TargetType="utu:Divider">
+	<ResourceDictionary.ThemeDictionaries>
+		<ResourceDictionary x:Key="Default">
+			<StaticResource x:Key="DividerForeground" ResourceKey="OnSurfacePressedBrush" />
+			<StaticResource x:Key="DividerSubHeaderForeground" ResourceKey="OnSurfaceLowBrush" />
+			<StaticResource x:Key="DividerSubHeaderStyle" ResourceKey="BodySmall" />
+			<x:Double x:Key="MaterialDividerHeight">1</x:Double>
+		</ResourceDictionary>
+		<ResourceDictionary x:Key="Light">
+			<StaticResource x:Key="DividerForeground" ResourceKey="OnSurfacePressedBrush" />
+			<StaticResource x:Key="DividerSubHeaderForeground" ResourceKey="OnSurfaceLowBrush" />
+			<StaticResource x:Key="DividerSubHeaderStyle" ResourceKey="BodySmall" />
```

Comment:
We don't usually extract styles as lightweight styling resources do we? @Xiaoy312 / @agneszitte

Reply:
yeah prefer to use like the other styles Foreground / FontSize / FontWeight ...